app/_components/commodities/Form.tsx

```diff
@@ -226,8 +240,8 @@ const Form: React.FC<FormProps> = ({ onSubmitHandler, isLoading }) => {
                 !isBuying &&
                 (errors.minDemand.message as string)}
             </FormErrorMessage>
-          </Stack>
-        </FormControl>
+          </FormControl>
```

Comment:
![image](https://github.com/ed-pilots-network/frontend/assets/50422789/41390c75-55eb-4848-8efd-c6ee886d1e5f)
The form is now completely broken on mobile

Reply:
Will fix this and push an update. Thanks, should have tested on mobile for this one.

Reply:
Fixed responsive views - 

![screenshot-Desktop-Chrome-480x720-dark](https://github.com/ed-pilots-network/frontend/assets/1340144/f2a30587-ca2d-42eb-a5c4-8b49cc6a509a)
![screenshot-Desktop-Chrome-768x1024-dark](https://github.com/ed-pilots-network/frontend/assets/1340144/83eb1fae-0647-41b6-9eac-22e9b2825c57)
![screenshot-Desktop-Chrome-1280x1024-dark](https://github.com/ed-pilots-network/frontend/assets/1340144/664412d3-e415-4443-935f-22289bab3f8a)
![screenshot-Desktop-Chrome-1920x1080-dark](https://github.com/ed-pilots-network/frontend/assets/1340144/338f0a56-e3da-4cb7-ab40-752e6f181921)

Reply:
I personally think that the subtitle should be on a newline, or maybe aligned to the right.

Reply:
Takes up a lot of empty space to move it to next line tbh. We can do a design discussion in discord about using taglines e.g. systems page doesn't use it for now.